lib/perl/Genome/InstrumentData/Solexa.pm

```diff
@@ -745,14 +745,15 @@ sub _convert_trimmer_to_sx_commands {
                 return;
             }
         } elsif ( $trimmer_params =~ /--\S+?[= ]\S+/ ) {
-            while($trimmer_params =~ /--(\S+)?[= ](\S+)/g) {
```

Comment:
Is it okay that the old version supported `--foo bar` and `--foo=bar` but the new version only supports `--foo bar`?